src/Lunr/Vortex/FCM/FCMApnsPayload.php

```diff
@@ -114,13 +113,14 @@ public function set_mutable_content(bool $mutable): static
      */
     public function set_priority(string $priority): static
     {
-        $priority = strtoupper($priority);
+        $matching = array_filter(
+            APNSPriority::cases(),
+            fn($case) => strtolower($case->name) == strtolower($priority)
+        );
```

Comment:
> It is also possible to use the [defined()](https://www.php.net/manual/en/function.defined.php) and [constant()](https://www.php.net/manual/en/function.constant.php) functions to check for the existence of or read an enum case if the name is obtained dynamically.

See https://www.php.net/manual/en/language.enumerations.basics.php

Reply:
Then you need to get the casing exactly right though. Or we need to reconstruct the string. Not sure what's the better approach.

Reply:
We have a casing format for enums, right? In this case it doesn't matter much but I think using `defined` or `constant` for this is a better practice

Reply:
> We have a casing format for enums, right?

Does Lunr have that?